unit-tests/repository_test.py

```diff
@@ -74,7 +74,21 @@ def test_extract_from_svn(self, repo: Tuple[Path, Path], tmp_path: Path):
         process = Popen(command)
 
         test_unit = SubversionRepo('svn://localhost/trunk')
-        test_unit.extract(tmp_path)
+        #
+        # It seems there can be a delay between the server starting and the
+        # listen socket opening. Thus we have a number of retries.
+        #
+        # TODO: Is there a better solution such that we don't try to connect
+        #       until the socket is open?
```

Comment:
I had a read through the SVN documentation and can't see anything obvious

Reply:
It's the classic problem with running it as a daemon. By definition it disconnects itself from the terminal and returns control to the launching process before it is finished setting up.